### ISSUE TITLE
Failed to create indexes on the PostgreSQL target 

### ISSUE THREAD

**Issue by slotix** [slotix/dbconvert-streams-public#26]
## Summary: 
Encountered an error while attempting to create indexes on the PostgreSQL target. 

Additionally, I have observed that the table name is duplicated in the index name.

## Error Message:
```
[2024/02/14 16:08:33] [error] dbs-target-writer/main.go:242 [target] ERROR: relation
"IDX_report_material_editions_report_material_editions_report_ty" already exists (SQLSTATE 42P07)
```

**Comment by slotix** [comment]
The problem seems to be attributed to the limitation of the [PostgreSQL "identifier length"](https://www.postgresql.org/docs/current/limits.html), which cannot exceed 63 bytes.

**Comment by slotix** [comment]
So, the problem has been resolved in version 0.10.4

1. The table name is no longer duplicated in the index.
2. If the name exceeds 64 characters, an error is thrown.

Download the latest binaries from:

| Operating System | Architecture | Link                                                                                |
| ---------------- | ------------ | ----------------------------------------------------------------------------------- |
| Linux            | AMD64        | [Download](https://dbconvert.com/downloads/dbs/v0.10.4/dbs-v0.10.4-linux-amd64.zip)   |
| Mac OS X         | AMD64        | [Download](https://dbconvert.com/downloads/dbs/v0.10.4/dbs-v0.10.4-darwin-amd64.zip)  |
| Windows          | AMD64        | [Download](https://dbconvert.com/downloads/dbs/v0.10.4/dbs-v0.10.4-windows-amd64.zip) |
| Linux            | ARM64        | [Download](https://dbconvert.com/downloads/dbs/v0.10.4/dbs-v0.10.4-linux-arm64.zip)   |
